service/qos/service_level_controller.cc

```diff
@@ -103,63 +107,65 @@ future<> service_level_controller::update_service_levels_from_distributed_data()
         return make_ready_future();
     }
 
-    return container().invoke_on(global_controller, [] (service_level_controller& sl_controller) {
-        return with_semaphore(sl_controller._global_controller_db->notifications_serializer, 1, [&sl_controller] () {
-            return async([&sl_controller] () {
```

Comment:
Note to self: this thread is created every time an update is needed (in a seastar::repeat loop), so perhaps it's better off as a coroutine. Alternatively, the whole loop can be moved to a thread/coroutine

Reply:
2c4979fb59197fd175bec054eafe505c45f62335, but I'll only post it after this PR (or some future iteration of it) gets merged and next is fixed